simulation/getrecommendation.js

```diff
@@ -0,0 +1,19 @@
+const axios = require('axios');
+
+const getRecommendation = (userId) => {
+  axios.get(`http://127.0.0.1:3000/api/v1/recommendations/user/${userId}`)
+    .then((response) => {
+      console.log(response.data[0]);
```

Comment:
You might want to validate this data before you use it, to protect yourself from bad data and h4x0rz.